myproject/views.py

```diff
@@ -0,0 +1,227 @@
+from django.db.utils import IntegrityError
+from django.shortcuts import render
+from django.http import HttpResponse
+from django.views.decorators.csrf import csrf_exempt
+from django.views.decorators.http import require_http_methods
+from rest_framework.views import APIView
+import json
+from myproject.models import *
+from .serializers import *
+
+
+def invalid_request(func):
+
+    def wrapper(*args, **kwargs):
+        try:
+            return func(*args, **kwargs)
+        except IntegrityError as error:
+            if 'UNIQUE' in error.args[0]:
+                result_text = "Record already exists for the given user and project"
+            else:
+                result_text = "Database constraints violated"
+
+            response = {
+                'result': result_text,
+                'status_code': 500,
+                'message': 'Internal Server Error'
+            }
+            return HttpResponse(json.dumps(response))
+
+        except Exception as error:
+            response = {
+                'result': error.args[0],
+                'status_code': 500,
+                'message': 'Internal Server Error'
+            }
+            return HttpResponse(json.dumps(response))
+
+    return wrapper
+
+
+# Create your views here.
+class user_views(APIView):
```

Comment:
The `views` class name `user`Classes naming should be camel cased. In this case `UserViews`